ElyaHovhannisyan/func 12

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+float func (float * a, float *b)
+{
+   float p1=*a,p2=*b;
+    *a=p1+p2;
+    *b=p1-p2;
+    p1=p1*p2;
+    return p1;
+}
+
+int main()
+{
+    float x=2.5,y=3,z;
+    z=func(&x,&y);
```

Comment:
Changing the value of input parameters is not considered a good programming style. It is considered to be error-prone. However, for the scope of this problem solution, this is all right.